src/main/kotlin/carracing/game/domain/data/Car.kt

```diff
@@ -3,12 +3,34 @@ package carracing.game.domain.data
 private const val INITIAL_POSITION = 0
 private const val MIN_NUMBER_TO_MOVE = 4
 
-data class Car(
-    var position: Int = INITIAL_POSITION,
+class Car private constructor(
+    var position: Int,
```

Comment:
Please take a look at the comment below and consider it in your implementation:
https://github.com/next-step/kotlin-racingcar/pull/1809#discussion_r2030103887